proto/query.proto

```diff
@@ -628,6 +628,5 @@ message TransactionMetadata {
   string dtid = 1;
   TransactionState state = 2;
   int64 time_created = 3;
-  int64 time_updated = 4;
```

Comment:
How about either adding a reserved for 4, or maybe we should make the next one 4, as it doesn't need backwards compatibility?